StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/BottomSheetViewController.swift

```diff
@@ -90,14 +90,23 @@ class BottomSheetViewController: UIViewController, BottomSheetPresentable {
             // First, get the old height of the content + navigation bar + safe area.
             manualHeightConstraint.constant = oldContentViewController.view.frame.size.height + navigationBarContainerView.bounds.size.height + view.safeAreaInsets.bottom
 
+            // Take a snapshot of the old content and add it to our container - we'll fade it out
+            let oldView = oldContentViewController.view!
+            let renderer = UIGraphicsImageRenderer(size: oldView.bounds.size)
+            let image = renderer.image { _ in
+                oldView.drawHierarchy(in: oldView.bounds, afterScreenUpdates: false)
```

Comment:
Might want to look into snapshotView() for performance, but I'm not sure if it matters these days for this use case

Reply:
Ooh nice thanks!